backend/project-service/api/src/main/kotlin/Projects.kt

```diff
@@ -195,6 +195,13 @@ typealias UpdateDataManagementPlanResponse = Unit
 typealias FetchDataManagementPlanRequest = Unit
 data class FetchDataManagementPlanResponse(val dmp: String?)
 
+data class SearchProjectPathsRequest(
+    val path: String
+)
+data class SearchProjectPathsResponse(
+    val paths: Map<String, String>
```

Comment:
It is not obvious from the name what this is a map between. I think something like `projectIdToPath` would be better.

But since this is now a bit more generalized search endpoint it might make sense if we include a bit more output data than just the title and ID. This would make this endpoint more useful in the future and would cut down on our super-specialized endpoints.